docs/source/contributing.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to Scylla Operator
+
+## Prerequisites
+
+To develop Scylla Operator, ensure your environment has the following items configured:
+
+1. [Go](https://golang.org/dl)
+   * Check `go.mod` for minimal version requirement.
+2. Git client
+3. GitHub account
```

Comment:
On the second thought, I would drop this one, there's no point explaining such basics.

Reply:
This is done now.